webapp/channels/src/components/admin_console/billing/billing_history.test.tsx

```diff
@@ -99,12 +97,20 @@ describe('components/admin_console/billing/billing_history', () => {
     const store = mockStore(state);
 
     test('should match snapshot', () => {
-        const wrapper = shallow(
+        renderWithIntl(
             <Provider store={store}>
                 <BillingHistory/>
             </Provider>,
         );
-        expect(wrapper).toMatchSnapshot();
+
+        expect(screen.queryByText('Billing History')).toBeInTheDocument();
+        expect(screen.queryByText('Transactions')).toBeInTheDocument();
+        expect(screen.queryByText('All of your invoices will be shown here')).toBeInTheDocument();
```

Comment:
What you are doing with these migrations is fantastic and will help us in going off enzyme later on

Reply:
^ this

Reply:
I totally agree!